### PR TITLE
Added HealInfo for gd1 and gd2

### PR DESCRIPTION
**Pull request by vredara**
added healinfo_gd1.go and healinfo_gd2.go to handle
respective heal info

Signed-off-by: venkata edara <redara@redhat.com>

### CODE REVIEW

File: pkg/glusterutils/healinfo_gd1.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+import (
+	"encoding/xml"
+	"fmt"
+	"strconv"
+)
+
+// HealInfo gets gluster vol heal info (GD1)
+func (g GD1) HealInfo(vol string) ([]HealEntry, error) {
+	cmd := fmt.Sprintf("gluster vol heal %s info --xml", vol)
```
Comment (aravindavk) [inline pkg/glusterutils/healinfo_gd1.go]: use `g.config.GlusterCmd`

```
fmt.Sprintf("%s vol heal %s info --xml", g.config.GlusterCmd, vol)
```

File: pkg/glusterutils/types.go [hunk 2]
```diff
@@ -96,12 +96,20 @@ type Volume struct {
 	ReplicaCount            int               `json:"replica-count"`
 }
 
+// HealEntry describe gluster heal info for each brick
+type HealEntry struct {
+	HostUUID       string
```
Comment (aravindavk) [inline pkg/glusterutils/types.go]: Change this to `PeerID`

File: pkg/glusterutils/healinfo_gd1.go (hunk 1 above)
Comment (vredara) [reply pkg/glusterutils/healinfo_gd1.go]: Yes  Brickpath is available. The Name in HealEntry struct is of hostname:<brick_path>

Comment (aravindavk) [reply pkg/glusterutils/healinfo_gd1.go]: nice, then we can split into two separate fields. `Hostname` and `Brick`

**Comment by aravindavk** [comment]
Updates: #53

**Comment by vredara** [comment]
Added Hostname and Brick to HealEntry struct

**Pull request merged**